src/plugin.ts

```diff
@@ -1,14 +1,13 @@
 import { Octokit } from "@octokit/rest";
 import { PluginInputs } from "./types";
 import { Context } from "./types";
-import { askQuestion } from "./handlers/ask-llm";
-import { addCommentToIssue } from "./handlers/add-comment";
-import { LogLevel, LogReturn, Logs } from "@ubiquity-dao/ubiquibot-logger";
+import { LogLevel, Logs } from "@ubiquity-dao/ubiquibot-logger";
```

Comment:
This package name doesnt seem right as we officially deprecated ubiquibot some weeks ago

Reply:
@0x4007 We should delete these to avoid confusion, they are still accessible within NPM.

Reply:
I cant now hope to in some days.